prov/rxm/src/rxm_conn.c

```diff
@@ -186,6 +195,14 @@ static void rxm_conn_connected_handler(struct util_cmap_handle *handle)
 {
 	struct rxm_ep *rxm_ep = container_of(handle->cmap->ep, struct rxm_ep, util_ep);
 	struct rxm_conn *rxm_conn = container_of(handle, struct rxm_conn, handle);
+	struct rxm_tx_entry *tx_entry;
+
+	while (!dlist_empty(&rxm_conn->tx_entry_for_release)) {
+		dlist_pop_front(&rxm_conn->tx_entry_for_release, struct rxm_tx_entry,
+				tx_entry, postponed_entry);
+		rxm_tx_buf_release(rxm_ep, tx_entry->tx_buf);
+		rxm_tx_entry_release(&rxm_conn->send_queue, tx_entry);
+	}
```

Comment:
This duplicates the same code as above.

Reply:
Is rxm_conn_connected_handler called from the main thread?